src/api/search/src/routes/query.js

```diff
@@ -0,0 +1,17 @@
+const { Router, logger } = require('@senecacdot/satellite');
+const { validateQuery } = require('../bin/validation');
+const search = require('../search');
+
+const router = Router();
+
+router.get('/', validateQuery, async (req, res) => {
+  try {
+    const { text, filter, page, perPage } = req.query;
+    res.send(await search(text, filter, page, perPage));
+  } catch (error) {
+    res.status(500).send(`There was an error while executing your query: ${error}`);
```

Comment:
For the sake of using APM, you might want to call `next(err)` instead, and let it go down through to the error handler, where we'll log and deal with it in APM.

Reply:
Done